packages/eslint-plugin/docs/rules/no-misused-spread.md

```diff
@@ -0,0 +1,100 @@
+---
+description: 'Disallow using the spread operator when it might cause unexpected behavior.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-misused-spread** for documentation.
+
+The [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (`...`) is a powerful feature in JavaScript that can be misused in ways not always detectable by TypeScript. This rule disallows using the spread operator on types where spreading can lead to unexpected behavior.
+
+## Examples
+
+<!--tabs-->
```

Comment:
[docs] The documentation is now being built with Docusaurus v3: https://github.com/typescript-eslint/typescript-eslint/pull/8209. Could you please update these `<!--tabs-->` comments to `<Tabs>` MDX tags?

Reply:
Done